src/assets/sass/components/_chart.scss

```diff
@@ -30,6 +30,29 @@ ccl-chart {
     padding-top: 1.5rem;
   }
 
+  .chart-legend {
```

Comment:
Can any of the colors in this newly added CSS block use the colors defined in `_variables.scss` or be tweaked slightly to use those colors or variations of them?